.github/workflows/docker.yaml

```
@@ -0,0 +1,112 @@
+#
+# GitHub actions for building and testing.
+#
+# For best support, use `-latest` for runners spinning up containers. More at
+# https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners.
+
+name: Docker
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# Set to 'yes' to open a tunnel to GitHub's VMs through tmate on failures.
+# Also increase timeout-minutes for the relevant OS when debugging remotely.
+env:
+  CHEVAH_DEBUG: 'no'
+
+# Using a job name that doesn't contain the OS name, to minimize the risk of
+# confusion with the OS names of the containers, which are the relevant ones.
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    strategy:
+      fail-fast: false
+      matrix:
+        container: [ 'alpine:3.12', 'centos:8.2.2004', 'centos:5.11' ]
+    timeout-minutes: 30
+    steps:
+
+    # To avoid duplicating the other steps, setup steps are allowed to fail.
+    # First 'run' command should only run successfully on the intended target.
+    # OpenSSL gets updated by apk, but that is the Alpine way, so it's fine.
+    - name: Alpine 3.12 setup
+      continue-on-error: true
```

I think that skipping the step is a more elegant solution.

```suggestion
      if:  matrix.container == 'alpine:3.12`
```